clients/docs/go.rst

```diff
@@ -0,0 +1,134 @@
+.. _client-examples-go:
+
+Go
+===
+
+In this tutorial, you will run a Golang client application that produces
+messages to and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  A functioning Go environment with the `Confluent Golang Client
+   for Apache Kafka <https://github.com/confluentinc/confluent-kafka-go>`__
+   installed.
+
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. Clone the `confluentinc/examples GitHub repository
+   <https://github.com/confluentinc/examples>`__ and check out the
+   :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+      git clone https://github.com/confluentinc/examples
+      cd examples
+      git checkout |release|-post
+
+#. Change directory to the example for Go.
+
+   .. code-block:: bash
+
+      cd clients/cloud/go/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Basic Producer and Consumer
+---------------------------
+
+In this example, the producer writes Kafka data to a topic in your Kafka
+cluster. Each record has a key representing a username (for example, ``alice``)
+and a value of a count, in JSON format (for example, ``{"Count": 0}``). The
+consumer reads the same topic and keeps a rolling sum of the counts as it
+processes each record.
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Run the producer, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka cluster
+   - topic name
+
+   .. code-block:: bash
+
+      go build producer.go
```

Comment:
@tikimims I'm sorry I told you the wrong information yesterday.  The line below is required, it should follow immediately after the build

```
./producer -f $HOME/.confluent/librdkafka.config -t test1		
```